backend/src/api/routes/webscraper/index.routes.ts

```
@@ -0,0 +1,99 @@
+    try {
+      await service.disconnectApify();
+      res.status(204).send();
+    } catch (err) {
+      next(err);
+    }
+  }
+);
+
+function parseLimit(raw: unknown, fallback: number, max: number): number {
+  const n = parseInt(String(raw ?? fallback), 10);
+  if (!Number.isFinite(n) || n <= 0) return fallback;
```

ESLint

[error] Expected { after 'if' condition.

(curly)